src/main/kotlin/com/re/back/exceptions/CustomException.kt

```diff
@@ -0,0 +1,3 @@
+package com.re.back.exceptions
+
+abstract class CustomException(val statusCode: Int, message: String) : RuntimeException(message)
```

Comment:
I recommend make the name of the exp more explained, maybe something like `AlreadyRegisterdException`

Reply:
I make this a generic/abstract exception, and any specific exception will extend this exception type.
Actually I make this generic exp type to make it more standard in the context of exception handling.

**Example:**
- `AlreadyRegisteredException` & `NotMatchedPasswordException` are subtypes of `CustomException`; different in statusCode and message, but both have the same way of handling and same fields.